### PR TITLE
mock VM container and usage

### PR DESCRIPTION
**Pull request by meiertgrootes**
Please check whether the instructions to build and launch the container with the mock VM image are intelligible and if you spot any issues with the Dockerfile

### CODE REVIEW

**Review by rogerkuou — APPROVED** [review]

File: Dockerfile_manpy37 [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    openssl \
+    libffi-dev \
+    lzma \
+    liblzma-dev \
+    python3-dev \
+    python3-setuptools \
+    wget \
+    curl \
+    mc \
+    ssh
```
Comment (fnattino) [inline Dockerfile_manpy37]: Missing trailing backslash:
```suggestion
    ssh \
```

File: Dockerfile_manpy37 [hunk 2]
```diff
@@ -0,0 +1,71 @@
+FROM ubuntu:16.04
```
Comment (fnattino) [inline Dockerfile_manpy37:1]: Maybe rename it to `Dockerfile`? In the end it's the only docker file..

File: cluster_cmd [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# The following commands can be issued at a python prompt to setup a
```
Comment (fnattino) [inline cluster_cmd]: One could also run this as a python script, so maybe add `.py` extension?

**Review by fnattino — APPROVED** [review]
I followed the steps you describe and I was able to spin up the docker-nodes cluster with little effort - very cool!! Just two minor things, then definitely merge!

File: Dockerfile_manpy37 (hunk 1 above)
Comment (meiertgrootes) [reply Dockerfile_manpy37]: done

File: Dockerfile_manpy37 (hunk 2 above)
Comment (meiertgrootes) [reply Dockerfile_manpy37:1]: Done

File: cluster_cmd (hunk 3 above)
Comment (meiertgrootes) [reply cluster_cmd]: Running this as a python script would set up a cluster with a scheduler, but the client would be lost as soon as the script exits. The Script would nit, however, provide any information on where to reach the scheduler.

If one wants to accomplish this, setting up a cluster from the command line with a script, the better option would be to use the disk-ssh command line script, which is installed along with dask.

The purpose of this, was to have quick reference for the relevant commands from within a pythin script or module.

**Comment by meiertgrootes** [comment]
merged #2 into master

**Pull request merged**